tools/gcc/git(41):ERROR:105: Unable to locate a modulefile for 'tools/gcc/git/2.8.2'
              total        used        free      shared  buff/cache   available
Mem:         191862       19122      146094         133       26644      171873
Swap:          8191         858        7333
Fri May 21 23:34:17 2021       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 460.32.03    Driver Version: 460.32.03    CUDA Version: 11.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Tesla V100-SXM2...  Off  | 00000000:1A:00.0 Off |                    0 |
| N/A   52C    P0    73W / 300W |   2850MiB / 16160MiB |      0%   E. Process |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   1  Tesla V100-SXM2...  Off  | 00000000:1C:00.0 Off |                    0 |
| N/A   40C    P0    58W / 300W |  15588MiB / 16160MiB |     12%   E. Process |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   2  Tesla V100-SXM2...  Off  | 00000000:1D:00.0 Off |                    0 |
| N/A   40C    P0    56W / 300W |  15588MiB / 16160MiB |     11%   E. Process |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
|   3  Tesla V100-SXM2...  Off  | 00000000:1E:00.0 Off |                    0 |
| N/A   44C    P0    59W / 300W |      0MiB / 16160MiB |      0%   E. Process |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A    160245      C   python                            585MiB |
|    1   N/A  N/A     97936      C   python                          15585MiB |
|    2   N/A  N/A     98433      C   python                          15585MiB |
+-----------------------------------------------------------------------------+
2021-05-21 23:34:35.833277: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-21 23:35:34.972075: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-05-21 23:35:34.979728: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcuda.so.1
2021-05-21 23:35:36.098502: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:1e:00.0 name: Tesla V100-SXM2-16GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 15.78GiB deviceMemoryBandwidth: 836.37GiB/s
2021-05-21 23:35:36.098572: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-21 23:35:36.935902: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
2021-05-21 23:35:36.936102: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.10
2021-05-21 23:35:37.328438: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-05-21 23:35:37.434010: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-05-21 23:35:38.224594: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-05-21 23:35:38.348221: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.10
2021-05-21 23:35:39.649419: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.7
2021-05-21 23:35:39.654869: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-05-21 23:35:39.686963: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-05-21 23:35:39.691588: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:1e:00.0 name: Tesla V100-SXM2-16GB computeCapability: 7.0
coreClock: 1.53GHz coreCount: 80 deviceMemorySize: 15.78GiB deviceMemoryBandwidth: 836.37GiB/s
2021-05-21 23:35:39.691670: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-21 23:35:39.691750: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
2021-05-21 23:35:39.691794: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.10
2021-05-21 23:35:39.691834: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-05-21 23:35:39.691874: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-05-21 23:35:39.691913: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-05-21 23:35:39.691952: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.10
2021-05-21 23:35:39.691992: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.7
2021-05-21 23:35:39.696902: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-05-21 23:35:39.713336: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-21 23:35:51.496262: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1261] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-05-21 23:35:51.496826: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1267]      0 
2021-05-21 23:35:51.496866: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1280] 0:   N 
2021-05-21 23:35:51.531673: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 14761 MB memory) -> physical GPU (device: 0, name: Tesla V100-SXM2-16GB, pci bus id: 0000:1e:00.0, compute capability: 7.0)
2021-05-21 23:35:51.552016: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-05-21 23:35:55.006644: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)
2021-05-21 23:35:55.008090: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 2100000000 Hz
WARNING:tensorflow:AutoGraph could not transform <function custom_mae at 0x2b91bafb5820> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: module 'gast' has no attribute 'Index'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
WARNING:tensorflow:AutoGraph could not transform <function custom_crossentropy at 0x2b91bafb5790> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: module 'gast' has no attribute 'Index'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
2021-05-21 23:35:58.941002: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
2021-05-21 23:36:02.752834: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.7
2021-05-21 23:36:13.927641: W tensorflow/stream_executor/gpu/asm_compiler.cc:63] Running ptxas --version returned 256
2021-05-21 23:36:14.025832: W tensorflow/stream_executor/gpu/redzone_allocator.cc:314] Internal: ptxas exited with non-zero error code 256, output: 
Relying on driver to perform ptx compilation. 
Modify $PATH to customize ptxas location.
This message will be only logged once.
313/313 - 272s - loss: 4.1051 - presence_loss: 0.1743 - coordinates_loss: 0.3102 - letter_loss: 3.6205 - presence_accuracy: 0.9450
Epoch 1/10
313/313 - 178s - loss: 3.8711 - presence_loss: 0.1071 - coordinates_loss: 0.1687 - letter_loss: 3.5954 - presence_accuracy: 0.9699
Epoch 2/10
313/313 - 181s - loss: 3.8227 - presence_loss: 0.0899 - coordinates_loss: 0.1367 - letter_loss: 3.5961 - presence_accuracy: 0.9743
Epoch 3/10
313/313 - 171s - loss: 3.7928 - presence_loss: 0.0780 - coordinates_loss: 0.1208 - letter_loss: 3.5940 - presence_accuracy: 0.9754
Epoch 4/10
313/313 - 182s - loss: 3.7697 - presence_loss: 0.0681 - coordinates_loss: 0.1131 - letter_loss: 3.5886 - presence_accuracy: 0.9797
Epoch 5/10
313/313 - 180s - loss: 3.7673 - presence_loss: 0.0669 - coordinates_loss: 0.1068 - letter_loss: 3.5935 - presence_accuracy: 0.9810
Epoch 6/10
313/313 - 182s - loss: 3.7560 - presence_loss: 0.0619 - coordinates_loss: 0.1053 - letter_loss: 3.5888 - presence_accuracy: 0.9816
Epoch 7/10
313/313 - 175s - loss: 3.7496 - presence_loss: 0.0566 - coordinates_loss: 0.1042 - letter_loss: 3.5888 - presence_accuracy: 0.9845
Epoch 8/10
313/313 - 180s - loss: 3.7462 - presence_loss: 0.0571 - coordinates_loss: 0.1002 - letter_loss: 3.5889 - presence_accuracy: 0.9826
Epoch 9/10
313/313 - 179s - loss: 3.7413 - presence_loss: 0.0543 - coordinates_loss: 0.0991 - letter_loss: 3.5879 - presence_accuracy: 0.9830
Epoch 10/10
313/313 - 180s - loss: 3.7392 - presence_loss: 0.0552 - coordinates_loss: 0.0991 - letter_loss: 3.5849 - presence_accuracy: 0.9832
2021-05-22 00:10:27.552968: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
